node/hack/governor/src/index.ts

```diff
@@ -12,6 +12,8 @@ import { Connection, JsonRpcProvider } from "@mysten/sui.js";
 import { arrayify, zeroPad } from "ethers/lib/utils";
 
 const MinNotional = 0;
+// Price change tolerance in %. Fallback to 30%
+const PriceDeltaTolerance = process.env.PRICE_TOLERANCE ? parseInt(process.env.PRICE_TOLERANCE) : 30;
```

Comment:
Do you need to enforce a range e.g. [0, 100]?  Or it's up to the user to be smart because this is in the hack directory?

Reply:
Yeah I wasn't too worried about it since it's in the hack directory, but I've added a upper + lower limit just in case